From 242ae54530c89af8d081b2b6a00a7789794ec4d4 Mon Sep 17 00:00:00 2001
From: Naveen Singh <naveen@nestlabs.com>
Date: Mon, 30 Oct 2017 23:00:03 +0100
Subject: [PATCH 02/12] configure.ac: Remove hard check of ns_initparse

AC_CHECK_LIB was performed for ns_initparse in libresolv.so. For android
there is no libresolv.so. This sysmobol exists in libc.so.This change
removes that hard check.
---
 configure.ac | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index bd670180..6b89e365 100644
--- a/configure.ac
+++ b/configure.ac
@@ -193,10 +193,13 @@ AM_CONDITIONAL(PPTP_BUILTIN, test "${enable_pptp}" = "builtin")
 
 AC_CHECK_HEADERS(resolv.h, dummy=yes,
 	AC_MSG_ERROR(resolver header files are required))
-AC_CHECK_LIB(resolv, ns_initparse, dummy=yes, [
-	AC_CHECK_LIB(resolv, __ns_initparse, dummy=yes,
-		AC_MSG_ERROR(resolver library support is required))
+
+if (test "${android_compile}" = "no"); then
+	AC_CHECK_LIB(resolv, ns_initparse, dummy=yes, [
+		AC_CHECK_LIB(resolv, __ns_initparse, dummy=yes,
+			AC_MSG_ERROR(resolver library support is required))
 ])
+fi
 
 AC_CHECK_HEADERS([execinfo.h])
 AM_CONDITIONAL([BACKTRACE], [test "${ac_cv_header_execinfo_h}" = "yes"])
-- 
2.11.0

